test/runtests.jl

```diff
@@ -17,7 +17,7 @@ if HAS_EXTENSIONS && MODULE_CAN_BE_TYPE_PARAMETER
     import Metis
 end
 
-const RUN_JET_TESTS = VERSION >= v"1.9"
+const RUN_JET_TESTS = false#VERSION >= v"1.9"
```

Comment:
Is the new solution (removing "JET" from "extras" and `add`ing it only if `RUN_JET_TESTS`) ok? 
Couldn't figure out another way to add it without nightly failing since JET won't precompile on nightly...